StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -8,8 +8,54 @@
 import Foundation
 @_spi(STP) import StripeCore
 
-protocol FinancialConnectionsAPIClient {
+class FinancialConnectionsAPIClient {
```

Comment:
Random analysis of whether this is the "simplest possible solution"...

Since it doesn't sound too feasible to "scatter" the logic of whether to use the new publishable key, then we must combine it.

The place where to combine it is somewhere in the API client. But right now our API client is a protocol. We can't(?) centralize logic in a protocol.

So we turn protocol into into a class to centralize logic.

...then its just a matter of asking how we shape the centralization logic (which is a bit separate from the first fundamental question of whether we "must" create a class/wrapper for the API)